.ci.yaml

```diff
@@ -3324,13 +3324,14 @@ targets:
       task_name: large_image_changer_perf_ios
     scheduler: luci
 
-  - name: Mac_ios macos_chrome_dev_mode
+  - name: Mac macos_chrome_dev_mode
+    bringup: true
     recipe: devicelab/devicelab_drone
     presubmit: false
     timeout: 60
     properties:
       tags: >
-        ["devicelab", "ios", "mac"]
+        ["devicelab", "hostonly"]
```

Comment:
Could we keep the `mac` tag?

Reply:
What does that tag do? It's not on any of the other hostonly tests.
https://github.com/flutter/flutter/blob/5ce07af9ca3cfad2301a4e229b0d30522c82e3ea/.ci.yaml#L3490-L3500

Reply:
Let's skip it in this PR then. Created https://github.com/flutter/flutter/issues/104104 to add tag for all host only targets.

Reply:
Does modifying the tags cause churn on the SkiaPerf side. If so, if we could modify the tags only once, that would be preferable.

Reply:
No. Tags here will not affect skia perf. This is what controls the skia perf tags: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipe_modules/test_utils/api.py#171